backend/modules/extract/services/ports/strategies.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+import logging
+
+
+logger = logging.getLogger(_name_)
+
+class ExtractStrategy(ABC):
```

Comment:
If you do so, I would actually move it to domain folder (if we decide to keep manager in domain folder).

Precisely saying to domain/ports.

Please remember to add __init__.py with allowed import from this folder.

In this case:

```python
from .ports import AbstarctExport

__all__ = ["AbstarctExport"]
```